.github/workflows/periodic-pytorch-win-vs2019-cuda11-cudnn8-py3.yml

```diff
@@ -80,34 +80,34 @@ jobs:
     if: ${{ github.repository_owner == 'pytorch' }}
     runs-on: ubuntu-18.04
     env:
+      TEST_RUNNER_TYPE: windows.8xlarge.nvidia.gpu
```

Comment:
This seems weird, but a test failure told me to run `.github/scripts/generate_ci_workflows.py` and push :confused:

Reply:
ok, reverted.